### DISCUSSION TITLE
download stops near its end please help   -- screenshot inside

### DISCUSSION THREAD

**Discussion by hemazone** [community/community#136744] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

dear sir ,

well i tried to download ComfyUI_windows_portable_nvidia.7z    ( 1.47 GB ) 
and when it reached to the end of downloading it stop and asked me for username and password  
and no mutter i tried to give it   it won't start  and pop up again asking  username and password
i tried to download again  and its same results 
![Untitl1ed](https://github.com/user-attachments/assets/7f71431f-52c7-42c8-9cce-c7be22356185)

  
any help will be much appreciated

**Comment by Yackens** [comment]
It looks like you are having some issues authenticating yourself. Possible solutions:
- Check if the website or service where you are downloading the file from requires you to log in or have a specific account.
- Ensure that you are logged into the correct account on the site before downloading.
- Contact the website's support to see if you need special access credentials.

**Comment by ebndev** [comment]
Hi @hemazone, thanks for being a part of the GitHub Community!

You might want to check out [ComfyUi's forum](https://app.element.io/#/room/#comfyui_space:matrix.org) for support with this question.

Best of luck!
